cmd/documentation/config_templates/config_readme.tmpl

```diff
@@ -194,6 +194,34 @@ comm method and add in your contact list if available.
 },
 ```
 
+## Configure exchange websocket subscriptions
+
++ Websocket subscriptions provide a stream of data from an exchange.
+Whilst subscriptions are specific to each exchange, some common examples are: candles, orderbook, ticker and allTrades.
+You can configure any supported channels in your exchange, but most likely you just want to disable some of the defaults, or change the default intervals.
+You cannot configure an empty list of subscriptions. Instead disable the websocket feature itself with `exchange.features.enabled.websocketAPI=false`.
+
+See the section `exchange.features.enabled.subscriptions` for configuring subscriptions:
+
+```js
+"subscriptions": [
+ {
+  "channel": "ticker"
```

Comment:
Update with Enabled field and regen docs

Reply:
Thanks. Fixed and rebased, docs regenerated.

```
You cannot configure an empty list of subscriptions, instead set all of the subscriptions to enabled: false.
```

Fixed 65db0de9